Sorts/BitonicSort.js

```diff
@@ -0,0 +1,74 @@
+/*
+
+    * JS program for bitonic sort
+    * Bitonic Sort is a parallel sorting algorithm that works by dividing the
+    array into two parts, recursively sorting them, and then merging them in a
+    specific way.
+    * more information: https://en.wikipedia.org/wiki/Bitonic_sorter
+
+    */
+function compAndSwap (a, i, j, order) {
```

Comment:
Rather than taking an `order` parameter, this should take a comparison function.